proxies/sandbox/apiproxy/targets/sandbox.xml

```diff
@@ -1,23 +1,14 @@
 <TargetEndpoint name="sandbox">
-  <Description/>
-  <FaultRules/>
-  <Flows/>
-  <PostFlow name="PostFlow">
-    <Request/>
-    <Response/>
-  </PostFlow>
-  <PreFlow name="PreFlow">
-    <Request/>
-    <Response>
-      <Step>
-        <Name>AssignMessage.AddCors</Name>
```

Comment:
More of a learning question, rather than a review, but:
Why was proxies/sandbox/apiproxy/policies/AssignMessage.AddBaseUrlHeader.xml deleted?